components/01-basics/_colors.html

```diff
@@ -157,4 +157,166 @@ <h2>Color palette</h2>
     </div>
   </div>
 
+  <h2>Color pairings: text and icons</h2>
+  <p>The colors in this system are used in specific combinations. The background color determines the color of the text, link text, and icons.</p>
+
+  <div class="slab" style="padding: 1rem 0">
```

Comment:
`div class = "slab t-sans"` to match the rest of the page above. See [line 13](https://github.com/fecgov/fec-pattern-library/pull/105/files#diff-1f1d73c892597d73e1d99b98357500f3R13)